util/envDynawo.sh

```diff
@@ -1294,6 +1587,211 @@ deploy_dynawo_autocompletion() {
   $DYNAWO_HOME/util/autocompletion/deploy_autocompletion.sh $*
 }
 
+build_doc() {
+  if [ ! -d "$DYNAWO_HOME/documentation" ]; then
+    error_exit "$DYNAWO_HOME/documentation does not exist."
+  fi
+  cd $DYNAWO_HOME/documentation
+  bash dynawo_documentation.sh
+}
+
+clean_doc() {
+  if [ ! -d "$DYNAWO_HOME/documentation" ]; then
+    error_exit "$DYNAWO_HOME/documentation does not exist."
+  fi
+  cd $DYNAWO_HOME/documentation
+  bash clean.sh
+}
+
+open_doc() {
+  if [ -x "$(command -v xdg-open)" ]; then
+    xdg-open $DYNAWO_HOME/documentation/dynawoDocumentation/DynawoDocumentation.pdf
```

Comment:
Why not using DYNAWO_PDFVIEWER here?

Reply:
Not sure I understand why we use first xdg-open and then the specified pdf viewer. Might be better to swap

Reply:
I swapped